src/EnergyPlus/SetPointManager.cc

```diff
@@ -158,6 +158,11 @@ constexpr std::array<std::string_view, static_cast<int>(CtrlVarType::Num)> contr
                                                                                                 "MAXIMUMMASSFLOWRATE",
                                                                                                 "MINIMUMMASSFLOWRATE"};
 
+constexpr std::array<std::string_view, static_cast<int>(ControlStrategy::Num)> strategyNameUC = {
```

Comment:
`Names` plural.

Reply:
I updated the ones that were changed as part of this PR, I didn't make a sweeping change to all the enums in there.